Codeinwp/otter-blocks
# chore: prepare BF event

Closes #.


### Summary


Add functions for sales events like Black Friday.

### Screenshots 

![image](https://github.com/Codeinwp/otter-blocks/assets/17597852/cabc0e54-4054-47e0-a5fa-447190da24c3)

![image](https://github.com/Codeinwp/otter-blocks/assets/17597852/942b167e-e41e-491f-b60c-c876706921ca)


----

### Test instructions


1. Check if the banners appear with the correct link/URL. More about this here: https://github.com/Codeinwp/themeisle/issues/1575#issuecomment-1757321452
2. By using all three plugins (Neve, Otter, Feedzy), check the admin notice (the small one) priority. There should be notice only for the highest priority product. The order is Neve > Otter > Feedzy. More on this here: https://github.com/Codeinwp/themeisle/issues/1575#issuecomment-1761388729

The giant banners will have a dynamic text that shows the remaining time in the top-left corner. The start is set early to facilitate the test (it will be changed after the approval). ~The time interval set for testing is 2023-10-10 00:00:00 and 2023-11-27 23:59:00.~ If possible, try to change the server time to see if the banner is working correctly when the time is over. 

The PRs are:
- Neve: https://github.com/Codeinwp/neve/pull/4121
- Otter: https://github.com/Codeinwp/otter-blocks/pull/1922
- Feedzy: https://github.com/Codeinwp/feedzy-rss-feeds/pull/805


---- 

### Checklist before the final review
